lib/partitioner/go.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'cocaine'
+require 'set'
+require 'partitioner/dependency_analysis'
+
+module Partitioner
+  # This partitioner shard Go repos
```

Comment:
grammar nit: This partitioner *shards* Go repos

Reply:
Fixed.